app.config.json

```diff
@@ -63,6 +63,9 @@
     "androidStatusBar": {
       "barStyle": "dark-content",
       "backgroundColor": "#EEEEEE"
+    },
+    "experiments": {
+      "tsconfigPaths": true
```

Comment:
Allows for path aliasing to be  picked up and used throughout for both compilation and intellisense.